Packs/Forcepoint_Security_Management_Center/Integrations/Forcepoint_Security_Management_Center/Forcepoint_Security_Management_Center.yml

```
@@ -0,0 +1,155 @@
+category: Network Security
+commonfields:
+  id: Forcepoint_Security_Management_Center
+  version: -1
+configuration:
+- display: Server URL
+  name: url
+  required: true
+  type: 0
+- displaypassword: API Key
+  additionalinfo: The API Key to use for connection
+  name: credentials
+  required: true
+  hiddenusername: true
+  type: 9
+- defaultvalue: '8082'
+  display: Port
+  name: port
+  required: true
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: 'Will be added later'
+display: 'Forcepoint Security Management Center'
+name: Forcepoint Security Management Center
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: The name of the IP List to create.
+      isArray: false
+      name: name
+      required: true
+      secret: false
+    - default: false
+      description: The list addresses to create.
+      isArray: false
+      name: addresses
+      required: false
+    - default: false
+      description: The comment to add to the IP List.
+      isArray: false
+      name: comment
+      required: false
+    description: 'Creates an IP List.'
+    execution: false
+    name: forcepoint-smc-address-create
+    outputs:
+    - contextPath: ForcepointSMC.Address.Name
+      description: 'The name of the IPList'
+      type: String
+    - contextPath: ForcepointSMC.Address.Addresses
+      description: 'The list of addresses in the IPList'
+      type: Unknown
+    - contextPath: ForcepointSMC.Address.Comment
+      description: 'The comment of the IPList'
+      type: String
```

is the object created an IP list? if so, maybe it should be:
```suggestion
    - contextPath: ForcepointSMC.IPList.Name
      description: 'The name of the IP List.'
      type: String
    - contextPath: ForcepointSMC.IPList.Address
      description: 'An address in the IP List.'
      type: String
    - contextPath: ForcepointSMC.IPList.Comment
      description: 'An overview or a brief explanation regarding the IP List.
      type: String
```
@Moishymat wdyt?

---

@Moishymat The object in the API is called IP List, and we refer to it as Address

---

We decided to rename the object in xsoar to IP List to match the standard in FP @dantavori